cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -43,13 +45,13 @@ type CloudControllerManagerOptions struct {
 
 // NewCloudControllerManagerOptions creates a new ExternalCMServer with a default config.
 func NewCloudControllerManagerOptions() *CloudControllerManagerOptions {
-	componentConfig := cmoptions.NewDefaultControllerManagerComponentConfig(ports.InsecureCloudControllerManagerPort)
+	componentConfig := NewDefaultCloudControllerManagerComponentConfig(ports.InsecureCloudControllerManagerPort)
```

Comment:
if this is here in the package, we can get rid of the prefix and just call is NewDefaultComponentConfig. wdyt?

Reply:
agree